test/format/format_config.c

```diff
@@ -344,6 +344,9 @@ config_table(TABLE *table, void *arg)
              ".runs.rows to %d if realloc_exact or realloc_malloc has been automatically set",
           table->id, WT_MILLION);
     }
+    /* FIXME-WT-15565 Write prepared truncate operation to disk. */
+    if (GV(PRECISE_CHECKPOINT) && GV(OPS_PREPARE))
+        config_off(table, "ops.truncate");
```

Comment:
There should be a warning (using `WARN`) message here, if truncate is explicitly mentioned in the config, you can check using `config_explicit` function.

Reply:
This check seems like it should reside in `config_run` instead of `config_table`, the idea is that if at a global level precise checkpoints and prepare ops are turned on, then "at a global level" we should turn off truncate.

Reply:
done, moved this to `config_run`